package.json

```diff
@@ -134,7 +134,6 @@
     "@emotion/react": "11.9.3",
     "@emotion/server": "11.4.0",
     "@emotion/styled": "^11.10.0",
-    "@floating-ui/react-dom-interactions": "^0.10.1",
```

Comment:
Dependency should not be removed from main package.json

Reply:
the `@floating-ui/react` pkg just be used at `@mantine/core`. The main `package.json` should contain all workspaces dependencies?

Reply:
Yes, it should, otherwise documentation will not have access to these dependencies.